pontoon/static/js/utils/datetime.js

```diff
@@ -0,0 +1,32 @@
+
+export default class Datetime {
+
+    constructor (datetime, locale) {
+        this.datetime = new Date(datetime);
+        this.locale = locale || 'en-GB';
+    }
+
+    get date () {
+        return new Intl.DateTimeFormat(
+            this.locale, this.dateFormat).format(this.datetime);
+    }
+
+    get dateFormat () {
+        return {
+            day: 'numeric',
+            month: 'long',
+            year: 'numeric'}
+    }
+
+    get time () {
+        return new Intl.DateTimeFormat(
+            this.locale, this.timeFormat).format(this.datetime);
+    }
+
+    get timeFormat () {
+        return {
+            hour: '2-digit',
+            minute: '2-digit',
+            second: '2-digit'}
+    }
+}
```

Comment:
This is something that [`moment.js`](http://momentjs.com/) (for example) does very well. Why reimplement that logic?

Reply:
no real objection to that - i was just trying to replicate code from here https://github.com/mozilla/pontoon/blob/master/pontoon/base/static/js/table.js#L10

Reply:
> This is something that moment.js (for example) does very well. Why reimplement that logic?

What are the browser requirements for Pontoon? When browsers can natively handle the basic needs outlined here via the `Intl` API, IMHO it wouldn't be wise to pull in a big dependency such as moment.js.

Reply:
Browser requirements for Pontoon are non-existent at the moment. We're going to work on them alongside those of Fluent in the near future.

Reply:
@adngdb can we go with this for now then please, as its basically doing what we do now. We can update if we feel we need some lib to handle this

Reply:
Yeah, I don't care too much about that. I'm using `moment.js` for Translate.Next so I figured we might as well use it here as well, but it's not super important.

Reply:
> I'm using moment.js for Translate.Next

What's this and how does it affect Pontoon?

Reply:
its a proposed rewrite of the translation ui

Reply:
Thanks. Is there a place it can be checked out? Or where can I learn more about it?

Reply:
atm i think the discussion is mostly internal - there are some bugs on bugzilla - @adngdb is the person driving the update

ill see if i can find relevant bugs...

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1243129

Reply:
@julen At the moment there's a document you can read about it, that explains pretty well what Translate.Next is: https://docs.google.com/document/d/1HDtCUMeogo2kjD5aNjulnlxSaC5kr928w5VnyXiAN3o/edit#

I haven't made any public announcements yet, but that will come. I want to involve the community early in the process of designing this, but I haven't taken the time to do that just yet.

Reply:
Thanks @adngdb, that pretty much answers the questions I had with my localizer hat 👍 Will plan to have a deeper look at that in the near future.